src/argus_htmx/destinations/forms.py

```diff
@@ -0,0 +1,79 @@
+from django import forms
+from django.forms import ModelForm
+
+from argus.notificationprofile.models import DestinationConfig, Media
+from argus.notificationprofile.serializers import RequestDestinationConfigSerializer
+from argus.notificationprofile.media import api_safely_get_medium_object
+
+
+class DestinationFormCreate(ModelForm):
+    settings = forms.CharField(required=True)
+
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop("request", None)
```

Comment:
Add comment stating this is needed because the serializer needs access to the request object